assembl/static2/js/app/components/debate/common/postActions.jsx

```diff
@@ -106,10 +91,18 @@ class PostActions extends React.Component {
           <div
             className="post-action"
             onClick={() => {
-              return openSharePostModal(postId);
+              return openShareModal({
+                title: modalTitle,
+                routerParams: routerParams,
+                elementId: postId,
+                social: useSocial
+              });
             }}
           >
-            <OverlayTrigger placement={this.state.screenWidth >= MEDIUM_SCREEN_WIDTH ? 'right' : 'top'} overlay={shareTooltip}>
+            <OverlayTrigger
+              placement={this.state.screenWidth >= MEDIUM_SCREEN_WIDTH ? 'right' : 'top'}
```

Comment:
This comment isn't pertinent to just your PR, but to all use cases of places. This check isn't complete enough. It should be expanded to check for the item's `width` and `scrollLeft` properties against the screen width, and placed accordingly.

This would resolve issues such as https://bluenove.atlassian.net/browse/DEVAS-888

Reply:
You do not have to resolve it in this PR. But, be aware that to resolve that bug, these are the kinds of checks that you should be making for tooltip placement